templates/partials/preview-question.html

```diff
@@ -0,0 +1,81 @@
+{% from 'components/panel/_macro.njk' import onsPanel %}
+{% from 'macros/helpers.html' import format_paragraphs %}
+
+{% set answers = question.answers %}
+
+<div class="ons-summary__item">
+
+  <h3 class="ons-u-mt-m ons-u-fs-r--b">{{ question.title }}</h3>
+
+  {% set answers_length = answers|length %}
+
+  {% if question.descriptions %}
+    {% set descriptions = question.descriptions %}
+    {% set descriptions_length = descriptions|length %}
+
+    {% for description in descriptions %}
+      <div class="ons-question__description ons-u-mb-m">{{- description | safe -}}</div>
+    {% endfor %}
+  {% endif %}
+
+  {%- if question.guidance -%}
+    {% set contents = question.guidance.contents %}
+    {% call onsPanel({
+      "id": "question-guidance-" ~ question.id,
+      "classes": "ons-u-mb-m"
+    }) %}
+      {% set contents = question.guidance.contents %}
+      {% include 'partials/contents.html' %}
+    {% endcall %}
+  {% endif %}
+
+  {% for answer in answers %}
+
+    {%- if answer.guidance -%}
+        {% set contents = answer.guidance.contents %}
+        {% call onsPanel({
+          "id": "question-guidance-" ~ question.id,
+          "classes": "ons-u-mb-m"
+        }) %}
+          {% set contents = answer.guidance.contents %}
+          {% include 'partials/contents.html' %}
+        {% endcall %}
+    {% endif %}
```

Comment:
This is incorrect, answer guidance should be a twisy, not a panel. Can we not just reuse `answer-guidance.html`?

Reply:
Also on a question page, answer guidance comes after the answers, so this should probably be the same.